aten/src/ATen/native/DispatchStub.h

```diff
@@ -427,7 +483,14 @@ struct RegisterPRIVATEUSE1Dispatch {
   REGISTER_AVX2_DISPATCH(name, fn)                                             \
   REGISTER_VSX_DISPATCH(name, fn)                                              \
   REGISTER_ZVECTOR_DISPATCH(name, fn)                                          \
-  REGISTER_SVE256_DISPATCH(name, fn)
+  REGISTER_SVE512_DISPATCH(name, fn)                                          \
+  REGISTER_SVE256_DISPATCH(name, fn)                                          \
```

Comment:
nit: looks like alignment of the backslashes is off as compared to line 485; shouldn't lint complain?

Reply:
somehow lintrunner passed locally and also the CI didn't complain.
Updated the PR.